pkg/stanza/fileconsumer/internal/tracker/tracker_other.go

```diff
@@ -52,9 +53,10 @@ OUTER:
 
 // On non-windows platforms, we keep files open between poll cycles so that we can detect
 // and read "lost" files, which have been moved out of the matching pattern.
-func (m *Manager) postConsume() {
-	m.closePreviousFiles()
+func (t *Tracker) PostConsume() {
```

Comment:
Since this is purely concerned with managing the file sets, I think it's ok for it to be in the tracker, but let's rename it so that it's not related to consumption. Maybe `EndPoll` for now?